pennylane/circuit_drawer/representation_resolver.py

```diff
@@ -329,8 +329,11 @@ def operator_representation(self, op, wire):
         Returns:
             str: String representation of the Operator
         """
-        if isinstance(op, qml.measure.MeasurementProcess) and op.obs is not None:
-            op = op.obs
+        if isinstance(op, qml.measure.MeasurementProcess):
+            if op.obs is not None:
+                op = op.obs
+            else:
+                return "raw"   # when no observable is provided we perform a raw measurement
```

Comment:
For sure! I agree that basis makes more sense than 'raw', but it might still be ambiguous? (eg. which basis?) 
Another suggestion would be: 
'comp_basis'  -- for computational basis, although this might not be true for the CV devices 

I will commit 'basis' for now and maybe we can get some users or members of the team to comment on it?